alfalfa_worker/jobs/openstudio/lib/workflow/measures/alfalfa_python_environment/measure.rb

```diff
@@ -79,7 +79,7 @@ def run(ws, runner, user_args)
         python_paths.append(run_dir_match[1] + "/.venv/lib/python3.8/site-packages")
       end
 
-      i = 3
+      i = 4
```

Comment:
what's up here?

Reply:
idf format changed and added an additional line of configuration to the python search path idf object.